pytorch_lightning/core/lightning.py

```diff
@@ -1,3 +1,14 @@
+"""
+The LightningModule is the "system recipe." It groups the following in one file:
+    - computational system definition
```

Comment:
I think that it needs a blank line to be treated as a list, be aware that reStructuredText and MarkDown formatting is a bit different...
http://www.unexpected-vortices.com/doc-notes/markdown-and-rest-compared.html